tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/Exceptions/HttpException.cs

```diff
@@ -4,17 +4,43 @@
 using System.Net;
 using System.Runtime.Serialization;
 using System.Threading.Tasks;
-using Azure.Sdk.Tools.TestProxy.Store;
 
 namespace Azure.Sdk.Tools.TestProxy.Common.Exceptions
 {
-    public class GitProcessException : Exception
+    public class HttpException : Exception
```

Comment:
I see. I somehow reversed the contents of these classfiles, and you're correcting this here. Thanks!